zkevm-circuits/src/super_circuit.rs

```diff
@@ -315,20 +328,14 @@ impl<const MAX_TXS: usize, const MAX_CALLDATA: usize> SuperCircuit<Fr, MAX_TXS,
             .map(|(_, bytecode)| bytecode.bytes.len())
             .sum::<usize>();
         let k = k.max(log2_ceil(64 + bytecodes_len));
-        let k = k.max(log2_ceil(64 + num_rows_required_for_steps));
-        let k = k + 1;
+        let k = k.max(log2_ceil(64 + get_num_rows_required));
         log::debug!("super circuit uses k = {}", k);
 
-        let mut instance: Vec<Vec<Fr>> = (1..POW_RAND_SIZE + 1)
-            .map(|exp| vec![block.randomness.pow_vartime(&[exp as u64, 0, 0, 0]); (1 << k) - 64])
-            .collect();
-        // SignVerifyChip -> ECDSAChip -> MainGate instance column
-        instance.push(vec![]);
-
+        let aux_generator = <Secp256k1Affine as CurveAffine>::CurveExt::random(rng).to_affine();
```

Comment:
> Why use a constant value in `MOCK_RANDOMNESS`

It's because the Keccak circuit uses Horner's method to evaluate the random linear combination of hash, if we use instance column to mock the randomness, it will raise the degree to 33

>  and random value in `aux_generator` for randomness?

That's the design of `ecc` in `halo2wrong`, so a prover can assign a random point as auxiliary generator for underlying multiexp algorithm. I think somehow we can have it as a fixed value, but actually I don't fully understand the underlying algorithm. @kilic should know much better.